src/features/draft/commands/handlers/transactional/internal-create-row.handler.ts

```diff
@@ -99,6 +99,7 @@ export class InternalCreateRowHandler extends DraftHandler<
         meta: input.meta,
         hash: await this.hashService.hashObject(input.data),
         schemaHash: input.schemaHash,
+        publishedAt: input.publishedAt,
```

Comment:
let's add test for it

Reply:
like src/features/draft/commands/handlers/__tests__/transactional/internal-update-row.spec.ts